code/modules/awaymissions/maploader/writer.dm

```diff
@@ -54,96 +38,110 @@
 	var/total_timer = start_watch()
 	var/timer = start_watch()
 	log_debug("Reading turfs...")
+
 	// Read the contents of all the turfs we were given
 	for(var/pos_z in sw.z to ne.z)
 		for(var/pos_y in ne.y to sw.y step -1) // We're reversing this because the map format is silly
 			for(var/pos_x in sw.x to ne.x)
-				var/turf/test_turf = locate(pos_x,pos_y,pos_z)
+				var/turf/test_turf = locate(pos_x, pos_y, pos_z)
 				var/test_template = make_template(test_turf, flags)
 				var/template_number = templates.Find(test_template)
 				if(!template_number)
 					templates.Add(test_template)
 					template_number = templates.len
 				template_buffer += "[template_number],"
 				CHECK_TICK
-
 			template_buffer += ";"
-
 		template_buffer += "."
-	template_buffer_text = jointext(template_buffer,"")
+
+	template_buffer_text = jointext(template_buffer, "")
 	log_debug("Reading turfs took [stop_watch(timer)]s.")
 
 	if(templates.len == 0)
 		CRASH("No templates found!")
-	var/key_length = round/*floor*/(log(letter_digits.len,templates.len-1)+1)
+
+	var/key_length = round/*floor*/(log(letter_digits.len, templates.len - 1) + 1)
```

Comment:
Best to put this comment here behind the actual code. Or above it. Placing it in between... jeesh how odd